epub33/rs/index.html

```diff
@@ -303,11 +303,11 @@ <h2>Internationalization</h2>
 					language for the documents' contents, when applicable. This includes:</p>
 
 				<ul>
-					<li id="confreq-rs-xml-lang">the <a data-cite="xml#sec-lang-tag"><code>xml:lang</code> attribute</a>
+					<li id="confreq-rs-xml-lang" data-tests="#cnt-18n_xhtml_xml_lang">the <a data-cite="xml#sec-lang-tag"><code>xml:lang</code> attribute</a>
```

Comment:
There should only be one underscore in any test name — #spec-reference_test-disambiguation. Can you fix here and in the tests, please?

I will review the tests soon, but leading full-day training sessions this week, which alas leaves no spare time. Maybe Friday!